src/main/resources/application.yml

```diff
@@ -78,6 +78,7 @@ nakadi:
   auth:
     plugin:
       factory: org.zalando.nakadi.plugin.auth.DefaultApplicationServiceFactory
+  event.max.bytes: 1000000
```

Comment:
I went for the default value in Kafka, which is 1,000,000, but I'm happy to go for a real 1MB if you think it is better